index.html

```diff
@@ -353,21 +353,21 @@ <h6 id="skills-title">Skills</h6>
         </div>
       </div> 
     </section>
-    <section class="contact-form-container">
+    <footer class="contact-form-container">
         <div class="description">
           <h6 class="description-text">I'm always interested in hearing about new projects,
              so if you'd like to chat please get in touch
           </h6>
         </div>
         <div class="contact-form">
-          <form action="#" id="form">
-            <input type="text" name="fullname" id="fullname" placeholder="Full name">
-            <input type="email" name="email" id="email" placeholder="Email address">
-            <textarea name="textarea" id="textarea" cols="30" rows="10" placeholder="Enter text here"></textarea>
+          <form action="https://for,spree.io/f/{form_id}" id="form" method="post">
+            <input type="text" maxlength="30" name="fullname" id="fullname" placeholder="Full name" required>
+            <input type="email" name="email" id="email" placeholder="Email address" required>
+            <textarea name="textarea" maxlength="500" id="textarea" cols="30" rows="10">Enter text here</textarea>
```

Comment:
Kindly use a placeholder so that it will disappear once the field is on focus. Currently, the user has to delete the default text.

Please, also add a required attribute to the textarea field. Of course, you would agree that entering a note to you is necessary.

![image](https://user-images.githubusercontent.com/20554583/193379168-8ac56691-9d40-405e-8171-edaecc33cd8c.png)